config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root 'welcome#index'
+
+  get '/', to: 'sessions#new', as: :login
```

Comment:
I'm pretty sure both of these are pointing at the same place, right? which one do you want to use?